src/data/customer/Setting.js

```diff
@@ -68,9 +71,35 @@ export const getCustomerSettingsAsync = async (token) => {
 
     return camelize(CustomerSettings.data);
   } catch (e) {
-    return Promise.reject({
-      statusCode: e.response.status,
-      statusText: e.response.statusText,
+    return apiResponseErrorHandler(e);
+  }
+};
+
+/** Retrieve Customer's Settings (reduced)
+ * @param {string} token
+ * @param {int} customerId
+ * @param {Object} payload {identityId, transactionGroupId}
+ * @return {Promise} settings object
+ */
+export const showCustomerSettingsAsync = async (
+  token,
+  customerId,
+  payload = {}
+) => {
+  try {
+    const paramsString = convertObjectIntoURLString(payload);
+    const CustomerSettingsShow = await axios({
+      method: 'GET',
+      url: `${endpoints.CUSTOMER_SETTINGS}/${customerId}/${
+        paramsString ? `?${paramsString}` : ''
+      }`,
+      headers: {Authorization: token},
     });
+    return {
+      ...CustomerSettingsShow,
+      data: camelize(CustomerSettingsShow.data),
```

Comment:
I think u just need to return camelize(CustomerSettingsShow.data)

Reply:
@vdanylov try to use the same structure as we do on other functions as much as possible. Is there a particular reason you created this return structure?